src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -243,52 +248,42 @@ import 'emby-ratingbutton';
     }
 
     function showRemoteControl() {
-        import('appRouter').then(({default: appRouter}) => {
+        import('../appRouter').then(({default: appRouter}) => {
             appRouter.showNowPlaying();
         });
     }
 
     let nowPlayingBarElement;
     function getNowPlayingBar() {
         if (nowPlayingBarElement) {
-            return Promise.resolve(nowPlayingBarElement);
-        }
-
-        return new Promise(function (resolve, reject) {
-            Promise.all([
-                import('appFooter-shared'),
-                import('itemShortcuts'),
-                import('css!./nowPlayingBar.css'),
-                import('emby-slider')
-            ])
-            .then(([appfooter, itemShortcuts]) => {
-                const parentContainer = appfooter.element;
-                nowPlayingBarElement = parentContainer.querySelector('.nowPlayingBar');
-
-                if (nowPlayingBarElement) {
-                    resolve(nowPlayingBarElement);
-                    return;
-                }
+            return nowPlayingBarElement;
+        }
 
-                parentContainer.insertAdjacentHTML('afterbegin', getNowPlayingBarHtml());
-                nowPlayingBarElement = parentContainer.querySelector('.nowPlayingBar');
+            const parentContainer = appFooter.element;
+            nowPlayingBarElement = parentContainer.querySelector('.nowPlayingBar');
 
-                if (layoutManager.mobile) {
-                    hideButton(nowPlayingBarElement.querySelector('.btnShuffleQueue'));
-                    hideButton(nowPlayingBarElement.querySelector('.nowPlayingBarCenter'));
-                }
+        if (nowPlayingBarElement) {
+            return nowPlayingBarElement;
+        }
 
-                if (browser.safari && browser.slow) {
-                    // Not handled well here. The wrong elements receive events, bar doesn't update quickly enough, etc.
-                    nowPlayingBarElement.classList.add('noMediaProgress');
-                }
+        parentContainer.insertAdjacentHTML('afterbegin', getNowPlayingBarHtml());
```

Comment:
Since `getNowPlayingBar` became sync, we fail here (`setIsClear` is `undefined`) https://github.com/jellyfin/jellyfin-web/blob/5b5d6a45b093b36c9885cfab3a3fe1008575515b/src/components/nowPlayingBar/nowPlayingBar.js#L337
because slider hasn't yet been upgraded by custom element (Chrome has synced tree upgrade, Firefox seems to be async).

Anyway, we have async `webcomponents` (polyfill for old browsers), so we need to be aware of this.
We can make `getNowPlayingBar` async again (even with `setTimeout(..., 0)`) or force upgrade
```suggestion
        parentContainer.insertAdjacentHTML('afterbegin', getNowPlayingBarHtml());
        window.CustomElements.upgradeSubtree(parentContainer);

```